src/xunit.netcore.extensions/TargetFrameworkMonikers.cs

```diff
@@ -24,6 +24,7 @@ public enum TargetFrameworkMonikers
         Netcoreapp = 0x1000,
         Uap = 0x2000,
         UapAot = 0x4000,
-        NetcoreCoreRT = 0x8000
+        NetcoreCoreRT = 0x8000,
+        All = ~0
```

Comment:
While All make some sense here it doesn't make a lot of sense in the SkipOnFramework context. I wonder if we really need to expose anything more here. You can just blindly use 0.